src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -1450,7 +1450,7 @@ bool LibraryCallKit::inline_vector_convert() {
       // Since input and output number of elements are not consistent, we need to make sure we
       // properly size. Thus, first make a cast that retains the number of elements from source.
       // In case the size exceeds the arch size, we do the minimum.
```

Comment:
If you're going to remove the MIN2 you must also change this comment.

Reply:
Sure. I will remove that in next commit.